### PR TITLE
merge Login signup

### PR DESCRIPTION
**Pull request by piyushsh25**
https://deploy-preview-3--zevnon.netlify.app/docs/pages/sign-up.html
https://deploy-preview-3--zevnon.netlify.app/docs/pages/login.html
https://deploy-preview-3--zevnon.netlify.app/docs/pages/logout.html

### CODE REVIEW

File: docs/css/sign-up.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+.change-method {
+    color: rgba(0, 0, 255, 0.945);
+}
+
+
+/* other ways to signup - login */
+
+.footer-elements {
+    margin: var(--size16) var(--size0);
+    padding: 1%;
```
Comment (GuptaEesh) [inline docs/css/sign-up.css:53]: 1% wouldn't make any big change as such.
- Either you can remove it.
- or use px for values smaller than 1rem or 10px

**Review by GuptaEesh — COMMENTED** [review]
Clean code. Just added a suggestion. Good to merge.

File: docs/css/sign-up.css (hunk 1 above)
Comment (piyushsh25) [reply docs/css/sign-up.css]: done!

**Review by adith-a-danthi — APPROVED** [review]
Clean code. Just a suggestion, try to use semantic HTML tags like `header`, `nav`, `main` and `footer`.

**Comment by piyushsh25** [comment]
merging after two good comments.

**Pull request merged**